build/lib/common.sh

```diff
@@ -78,7 +142,16 @@ function build::gather_licenses() {
      -o -name 'LICENSE.code' -o -name 'LICENCE.md' \) | while IFS= read -r NAME; do mkdir -p "${outputdir}/$(dirname $NAME)" && cp  "$NAME" "${outputdir}/${NAME}"; done
 }
 
-function build::common::use_go_version() {
+function build::common::remove_go_path() {
+  # This is the path where the specific go binary versions reside in our builder-base image
+  export PATH=$(echo "$PATH" | sed -e "s/^\/go\/go[0-9\.]*\/bin://")
```

Comment:
This looks troublesome, can't we just put the version we want first? This won't work in my env, although I'm not sure that matters.

Reply:
yeah i was thinking about that too, we probably can. ill make another rev on this.